blocks/bricks/base.py

```diff
@@ -556,6 +582,7 @@ def __init__(self, name=None):
         self.name = name
 
         self.children = []
```

Comment:
I don't create an attribute. This calls the setter, so `self.children = []` results in `self._children = Children(self, [])` being called by the property setter. Then the next call to `self.children` will return the `self_children` attribute which is now set to a `Children` object. If I don't call the setter here, the first call to `self.children` will result in an `AttributeError`. It's better to call the setter than it is to set private attributes directly in the constructor IMHO.